<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_page</class>
 <widget class="QWidget" name="Edit_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background:rgb(19,20,20);

</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>224</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>8</y>
      <width>250</width>
      <height>216</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:rgb(19,20,20);

</string>
    </property>
    <widget class="QPushButton" name="color_btn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/1x/color.png);

background:rgba(19,20,20,0.9);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chang_btn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>8</y>
       <width>225</width>
       <height>154</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
